exercises/pangram/pangram_test.rb

```diff
@@ -79,6 +79,6 @@ def test_pangram_with_non_ascii_characters
 
   def test_bookkeeping
     skip
-    assert_equal 2, BookKeeping::VERSION
+    assert_equal 3, BookKeeping::VERSION
   end
 end
```

Comment:
We can ignore the changes in pangram. To check the previos version, we can:

`git checkout master exercises/pangram`.